keyboards/tmo50/keymaps/xerpocalypse/keymap.c

```diff
@@ -0,0 +1,57 @@
+/* Copyright 2019 funderburker
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define LT1_SPC     LT(1, KC_SPC)
+#define LT2_SPC     LT(2, KC_BSPC)
+#define UNDRSCR     S(KC_MINS)
```

Comment:
QMK has a keycode for this already.

https://github.com/qmk/qmk_firmware/blob/969dd8be562ad8d52822673e065f5d9947eb809d/quantum/quantum_keycodes.h#L569

```suggestion
```